atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/oracle/OracleDdlTable.java

```diff
@@ -140,6 +179,10 @@ private void throwForMissingOverflowTable() {
                 LoggingArgs.tableRef(tableRef));
     }
 
+    /**
+     * Creates the table for the given reference. If the table needs to be modified (i.e. to add an overflow column),
```

Comment:
Thanks for the comment! I would probably change `table needs to be modified` to `overflow column in the table schema is changed, the please ensure alterTable... is updated respectively` and maybe just add that comment as an inline one near the overflow column part?

Reply:
Met you half way, what do you think of that?

Reply:
Ah, I think we have 2 different opinions of what the comment should be for.
My goal with the comment was to highlight to an AtlasDB dev that if they should modify the schema for the overflow column, they must modify the code in alterTable... to match.

OTOH, what we have here is more an indication to an AtlasDB _client_ that there exists another method for altering the table :)